lib/haxe/src/org/apache/thrift/transport/TFileStream.hx

```diff
@@ -24,6 +24,9 @@ import haxe.io.BytesBuffer;
 import haxe.io.Input;
 import haxe.io.Output;
 
+#if js
+import js.node.Fs;
+#end
 
```

Comment:
> Well, the haxe by self is such

This is a code smell, no matter what "the haxe" thinks. But not blocking issue for the commit.

> anyway this file is mostly used for tests 

That's why it says "TTestFileStream"? No, wait ... it doesn't. Joking aside, why do you want to limit the usage artifically by saying "oh, this is only test stuff? I see a slass that may be used for other purposes.